fastlane/fastlane_1_intro.md

```diff
@@ -0,0 +1,115 @@
+# Fastlane iOS
+
+Requirements:
+- you have some prior knowledge about [codesigning an iOS app](../ios/ios_signing/ios_signing_1.md)
+- you have an Apple developer account, enrolled in the Developer program (you have access to iTunes Connect)
+
+Codesigning and deploying an app to the Apple Store is a fastidious, time-consuming, error-prone process.
+See [this entry in Stack-Overflow](http://stackoverflow.com/questions/21250510/generate-pem-file-used-to-setup-apple-push-notification) just to get an idea. ;)
+
+Don't worry, here comes [Fastlane](https://fastlane.tools/)!
+
+![Fastlane](https://raw.githubusercontent.com/fastlane/fastlane/master/fastlane/assets/fastlane_text.png)
+
+Fastlane is a toolbox (written in Ruby) that will allow you to automate all these tedious process.
+
+## Install it
+
+Install it with:
+```shell
+gem install fastlane
+```
+Don't worry if it takes a few minutes. A lot of goodness is installing.
+
+## Try it out
+
```

Comment:
Good for blog article, not sure if adequate for a book that serves as a reference. Need review by another peer